deltacat/tests/compute/test_compact_partition_rebase.py

```diff
@@ -274,16 +275,24 @@ def test_compact_partition_rebase_same_source_and_destination(
         compacted_delta_locator, storage_type=StorageType.LOCAL, **ds_mock_kwargs
     )
     actual_rebase_compacted_table = pa.concat_tables(tables)
```

Comment:
We can add one assertion if the result should yield multiple tables.

Reply:
If I understand correctly, the suggestion is to add an assertion on `len(tables)`?